src/interfaces/assistants_web/src/components/Shared/HotKeys/HotKeysProvider.tsx

```diff
@@ -45,26 +35,23 @@ export const HotKeysProvider: React.FC<HotKeysProviderProps> = ({ customHotKeys
     [isDialogOpen, close, open]
   );
 
-  customHotKeys.forEach(({ commands, action, options, dependencies }) => {
-    // eslint-disable-next-line react-hooks/rules-of-hooks
-    useHotkeys(
-      commands,
-      () => {
-        if (isDialogOpen) {
-          close();
-        }
-        action();
-      },
-      { enableOnFormTags: true, ...options },
-      dependencies
-    );
-  });
-
-  const dialogCustomActions: QuickAction[] = customHotKeys.map(({ name, commands, action }) => ({
-    name,
-    commands,
-    action,
-  }));
-
-  return <HotKeysDialog isOpen={isDialogOpen} close={close} customActions={dialogCustomActions} />;
+  hotKeys
+    .map((hk) => hk.quickActions)
+    .flat()
+    .forEach(({ commands, action, options, dependencies }) => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
not for now, but i think we should check a better way to register hooks outside of a forloop